pkg/authorization/static/static_test.go

```diff
@@ -134,6 +134,55 @@ func TestStaticAuthorizer(t *testing.T) {
 				authorizer.AttributesRecord{Verb: "get", Resource: "services", ResourceRequest: true},
 			},
 		},
+		{
+			name: "groupMatch",
+			config: []StaticAuthorizationConfig{
+				{User: UserConfig{Groups: []string{"admin", "editors"}}, Verb: "get", Resource: "namespaces", ResourceRequest: true},
```

Comment:
Nit: I know this is "idiomatic" to the previous test data, but I would prefer, having readable specs:

```suggestion
			config: []StaticAuthorizationConfig{{
			    User: UserConfig{
			        Groups: []string{"admin", "editors"}
			    },
			    Verb: "get",
			    Resource: "namespaces",
			    ResourceRequest: true,
			},
```